packages/dual/dual.0.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Dual numbers library"
+description: "This is an implementation of dual numbers in OCaml"
+maintainer: ["drjdnielsen@gmail.com"]
+authors: ["Jason D. Nielsen"]
+license: "MIT"
+homepage: "https://github.com/drjdn/ocaml_dual"
+bug-reports: "https://github.com/drjdn/ocaml_dual/issues"
+depends: [
```

Comment:
```suggestion
depends: [
 "ocaml" {>= "4.08"}
```
The `Float` module has been inteoduced “recently”, you need a lower bound on the supported ocaml version